scripts/qa_crowbarsetup.sh

```diff
@@ -821,51 +823,51 @@ function onadmin_prepare_cloud_repos()
         add_bind_mount \
             "${localreposdir_target}/${CLOUDLOCALREPOS}/sle-11-x86_64/" \
             "${targetdir}"
-        echo $CLOUDLOCALREPOS > /etc/cloudversion
     else
         rsync_iso "$CLOUDSLE11DISTPATH" "$CLOUDSLE11DISTISO" "$targetdir"
-        cat "$targetdir/isoversion" > /etc/cloudversion
     fi
-    echo -n "This cloud was installed on `cat ~/cloud` from: " | \
-        cat - /etc/cloudversion >> /etc/motd
-    echo $cloudsource > /etc/cloudsource
 
     if [ ! -e "${targetdir}/media.1" ] ; then
         complain 35 "We do not have cloud install media in ${targetdir} - giving up"
     fi
 
-    zypper rr Cloud
-    safely zypper ar -f ${targetdir} Cloud
-
     if [ -n "$TESTHEAD" ] ; then
         case "$cloudsource" in
             GM4)
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 ;;
             GM4+up)
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 addcloud4testupdates
                 ;;
             GM5)
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 addsles12testupdates
                 addcloud5pool
                 ;;
             GM5+up)
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 addsles12testupdates
                 addcloud5testupdates
                 addcloud5pool
                 ;;
             develcloud4)
-                addsp3testupdates
+                addsp3testupdates "sp3tup"
                 ;;
             develcloud5|develcloud6)
-                addsp3testupdates
+                if [ -n $want_sles12_admin ]; then
+                  addsp3testupdates
```

Comment:
i don't get this part, you dont send an argument `$1` so the mount doesn't happen or what is the logic behind?